haechan45th/src/services/userService.js

```diff
@@ -16,7 +20,13 @@ const signUp = async (name, email, password, profileImage) => {
         emailValidationCheck(email);
         pwValidationCheck(password);
```

Comment:
```suggestion
        await emailValidationCheck(email);
        await pwValidationCheck(password);
```
내부 로직이 복잡하거나 수행 시간이 오래걸리는 함수가 아니기 때문에 정상 동작에는 문제 없을 수 있으나, 보다 확실한 비동기 처리를 위해 await를 추가해주세요!